handlers/high-charts/src/server.ts

```diff
@@ -74,12 +78,24 @@ app.post("/handler", async (req, res) => {
         if (serie["data"] && serie["data"].length > 0) {
             const data = serie["data"];
             const supportedCharts = ["line", "area", "spline", "areaspline"];
+
             if (supportedCharts.indexOf(serie["type"]) != -1) {
-                // We can work with this
+                // Line charts, we can work with this
                 console.log("Length: " + data.length);
+                console.log(`Processing chart type: ${serie["type"]}`);
```

Comment:
Verbally confirmed! Should be good! Juliette will know more about this if there are any problems!

Reply:
> Question: Where do the console logs appear? I genuinely don't know! I ask because ideally we want to see the error messages pop up in the orchestrator so that when things run all of the error messages across preprocessors and handlers are in the same place.

Answer: They appear in its docker container log, via 
```sh
# in /var/docker/image
docker compose logs -f high-charts-handler
```